Sources/SwiftSyntaxBuilder/ClosureParamConvenienceInitializer.swift

```diff
@@ -0,0 +1,23 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+
+extension ClosureParam {
+  public init (leadingTrivia: Trivia = [], unexpectedBeforeName: ExpressibleAsUnexpectedNodes? = nil, name: String, unexpectedBetweenNameAndTrailingComma: ExpressibleAsUnexpectedNodes? = nil, trailingComma: Token? = nil) {
+    self.init(leadingTrivia: leadingTrivia,
+              unexpectedBeforeName: unexpectedBeforeName,
+              name: Token(tokenSyntax: .identifier(name)),
+              unexpectedBetweenNameAndTrailingComma: unexpectedBetweenNameAndTrailingComma,
+              trailingComma: trailingComma)
+  }
+}
```

Comment:
Just curious, do we use this anywhere? Also wdyt about removing the unexpected nodes here (since they add noise, may complicate overload resolution and are probably not used by anyone constructing nodes with the convenience initializer)?

Reply:
Left over, sorry about that